pkgs/misc/vim-plugins/update.py

```diff
@@ -71,6 +70,12 @@ def f_retry(*args: Any, **kwargs: Any) -> Any:
 
     return deco_retry
 
+def makeRequest(url: str) -> urllib.request.Request:
```

Comment:
Actually I forgot we are doing python here. Did too much rust/go lately:

```suggestion
def make_request(url: str) -> urllib.request.Request:
```